src/Node/FS/Sync.purs

```diff
@@ -246,17 +248,17 @@ rmdir file = mkEff $ \_ -> runFn1
 mkdir :: forall eff. FilePath
                   -> Eff (fs :: FS, err :: Exception | eff) Unit
 
-mkdir = flip mkdir' 777
+mkdir = flip mkdir' mkPerms (r <> w <> x) (r <> w <> x) (r <> w <> x)
```

Comment:
Shouldn't this be `flip mkdir' $ mkPerms ...`? I don't understand how this is working as is.